src/key/user.js

```diff
@@ -59,7 +59,10 @@ class User {
       if (privateKey.hasSameFingerprintAs(primaryKey)) {
         throw new Error('Not implemented for self signing');
       }
-      const signingKey = await privateKey.getSigningKey(undefined, undefined, undefined, config);
+      const signingKey = await privateKey.getSigningKey(
+        undefined, undefined, undefined,
+        { ...config, rejectPublicKeyAlgorithms: new Set(), minRsaBits: 0 }
+      );
```

Comment:
Mmm wouldn't that imply that DSA keys won't have verifiable users by default, for instance?

Reply:
No I think it means that third-party user certification signatures from DSA keys won't verify by default. Similarly to how message signatures from DSA keys won't verify by default. Self-signatures on user IDs on DSA keys will still verify by default (so that the key itself is valid - but you can't verify anything with it).